bundles/AdminBundle/config/templating_twig.yaml

```diff
@@ -0,0 +1,7 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    Pimcore\Twig\Extension\AdminExtension: ~
```

Comment:
I would define this in `services.yaml`..no need to create a separate file for one service.